pom.xml

```diff
@@ -186,7 +186,7 @@
                     <artifactId>maven-surefire-plugin</artifactId>
                     <version>${version.surefire.plugin}</version>
                     <configuration>
-                        <argLine>-Xmx1024m -Dfile.encoding=UTF-8</argLine>
+                        <argLine>-Xmx1024m -Dfile.encoding=UTF-8  --add-opens java.base/java.nio=ALL-UNNAMED</argLine>
```

Comment:
We need to add a note in the readme/doc of the Java library about java 17 compatibility (in Python it should be hidden in the binding/image).

The configuration here is a workaround: this option is necessary to _run_ the expl-arrow library and given that we do it only during the tests. Said differently the option here is enough but in reality every time the library is executed with JDK 17 we need this option

Reply:
Good point @danielezonca, I'll add it to the docs next.